exp/examine/local.go

```diff
@@ -0,0 +1,164 @@
+package examine
+
+import (
+	"os"
+	"strings"
+
+	"github.com/chanzuckerberg/fogg/errs"
+	"github.com/hashicorp/go-getter"
+	"github.com/hashicorp/terraform-config-inspect/tfconfig"
+	"github.com/spf13/afero"
+)
+
+const apiHostname = "https://registry.terraform.io"
+const apiVersion = "/v1"
+const resourceType = "/modules/"
+
+//**Local refers to any files located within your local file system**
+
+//GetLocalModules retrieves all terraform modules within a given directory
+//TODO:(EC) Define local and global modules OR rename the values
+func GetLocalModules(fs afero.Fs, dir string) ([]ModuleWrapper, error) {
+	_, err := os.Stat(dir)
+	if err != nil {
+		return nil, err
+	}
+
+	modules, err := getAllModules(fs, dir)
```

Comment:
```suggestion
	return getAllModules(fs, dir)
```
and then delete lines 28-31